packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -121,10 +192,12 @@ export default class Reachability {
 
     if (reachabilityData) {
       try {
-        const reachabilityResults = JSON.parse(reachabilityData);
+        const reachabilityResults: InternalReachabilityResults = JSON.parse(reachabilityData);
```

Comment:
Why not call getReachabilityResults instead of manually grabbing them and parsing again?

Reply:
because getReachabilityResults() returns the results without the internal stuff like `isVideoMesh` field that we need to read here